Create ui-c8y-1021-36-1-add-no-whitespaces-only-allowed-validation-to-firmware-software-and-configuration.md (#2899)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-36-1-add-no-whitespaces-only-allowed-validation-to-firmware-software-and-configuration.md

---
date: ""

> title: Add no whitespaces only allowed validation to firmware, software and configuration forms

Reviewer: ```suggestion
title: Add validation to firmware, software and configuration forms to disallow values with only whitespaces
```

> title: Add validation to firmware, software and configuration forms to disallow values with only whitespaces

Reviewer 2: ```suggestion
title: Submitting values with only whitespaces in firmware, software and configuration forms no longer allowed
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4325
version: 1021.36.1
---

> Add no whitespaces only allowed validation to firmware, software and configuration forms

Reviewer: ```suggestion
In Device management, it was possible to enter values (e.g. Description, Version, URL, etc.) consisting only of whitespaces for firmware, software and configuration assets in the corresponding repositories. To prevent this, additional validation has been added to the respective forms which disallows submitting such names. This change improves data quality and consistency for newly created firmware, software and configuration assets.
```

> In Device management, it was possible to enter values (e.g. Description, Version, URL, etc.) consisting only of whitespaces for firmware, software and configuration assets in the corresponding repositories. To prevent this, additional validation has been added to the respective forms which disallows submitting such names. This change improves data quality and consistency for newly created firmware, software and configuration assets.

Reviewer 2: ```suggestion
In the Device management application, it previously was possible to enter values (for example, in the **Description**, **Version**, **URL** fields) consisting only of whitespaces for firmware, software and configuration items in the corresponding repositories. To prevent this, additional validation has been added to the respective forms which disallows submitting such names. This change improves data quality and consistency for newly created firmware, software and configuration items.
```
